### PR TITLE
Create the registry folder if it doesn't exist

### PR DESCRIPTION
**Pull request by jsancio**
There is race between docker creating the volume/folder and the python
creating the folder. If docker created the folder before the script had
a change to create then just continue.

### CODE REVIEW

**Comment by jsancio** [comment]
This should resolve #907 and #922

File: scripts/local-universe.py [hunk 1]
```diff
@@ -73,18 +71,26 @@ def main():
         docker_artifacts = dir_path / pathlib.Path("registry")
         repo_artifacts = dir_path / pathlib.Path("universe/repo/packages")
 
+        # There is a race between creating this folder and docker run command
+        # creating this volume
+        os.makedirs(str(docker_artifacts), exist_ok=True)
```
Comment (jsancio) [inline scripts/local-universe.py]: This is the actual fix. The rest are formatting changes.

**Comment by cruhland** [comment]
LGTM!

**Comment by ryadav88** [comment]
@jsancio do we need to update https://docs.mesosphere.com/1.8/administration/installing/deploying-a-local-dcos-universe/ ?

**Comment by jsancio** [comment]
@ryadav88 This doesn't change the existing documentation.

**Pull request merged**